inlineplz/linters/ansiblelint.py

```diff
@@ -3,8 +3,23 @@
 from __future__ import unicode_literals
 
 from inlineplz.parsers.base import ParserBase
+from inlineplz.decorators import linter
+import sys
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import sys" should be placed before "from inlineplz.parsers.base import ParserBase" (wrong-import-order)`